### DISCUSSION TITLE
Any way to hide the title and it's container?

### DISCUSSION THREAD

**Discussion by onurusluca** [withastro/starlight#1132] [Feature Requests]
### What version of `starlight` are you using?

0.11.2

### What is your idea?
I can leave the title blank so it is hidden but the container and it's space is still there, is there a way to achieve this?

![image](https://github.com/withastro/starlight/assets/51825926/e9c89c47-2727-4633-b7a9-28c991684a56)

I tried to select the class names and hide them in custom.css but it is just horrible practice:
```
body>div>div>div>div>main>div:nth-child(1) {
    display: none;
}
```
_(yes this actually removes the title container lol)_

`template: splash` doesn't remove the title neither. 

Couldn't find it in the docs. 

### Why is this feature necessary?

To have a blank page.

### Participation

- [X] I am willing to submit a pull request for this feature.

**Comment by HiDeoo** [comment]
Without going the [component overrides](https://starlight.astro.build/guides/overriding-components/) route, another possible selector could be `.content-panel:has(h1#_top:not([data-page-title]))`.

This may also be related to https://github.com/withastro/starlight/discussions/1112 with a `template: custom` frontmatter option where the desired result is to have an empty page.

**Comment by mavam** [comment]
It would be nice to be able to just say `title: false` in the frontmatter.

**Comment by delucis** [comment]
I will note that `title` is required intentionally because it’s an accessibility and SEO issue not to have an `h1` on the page (and we use the `title` frontmatter elsewhere too, for example in `<title>` and other metadata or for generating sidebar labels).

But yeah, probably an option like #1112 may make sense — with `title` still being required in frontmatter to use for those metadata purposes and the emphasis on the user to ensure an `h1` is still present.
